flexible/websocket-jetty/pom.xml

```diff
@@ -15,7 +15,7 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  xsi:schemaLocation="http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
revert this line. This was correct. The syntax is weird but `xsi:schemaLocation` does need two URLs separated by space.